UtopiaCity/Controllers/Life/LifeController.cs

```diff
@@ -54,7 +60,7 @@ public IActionResult Edit(string id)
                 .Cast<EventType>()
                 .Select(x => new EventTypeItem { Id = (int)x, Name = x.ToString() });
 
-            var selected = eventTypes.FirstOrDefault(x => x.Id.Equals((int)e.EventType));
+            var selected = Enumerable.FirstOrDefault<EventTypeItem>(eventTypes, (Func<EventTypeItem, bool>)(x => (bool)x.Id.Equals((int)(int)e.EventType)));
```

Comment:
double cast to int?

Reply:
Casting my enums to list of objects then finding selected object.
No idea where to cut my code.

Reply:
removed double cast
now this changes outdated